mycroft/skills/phillips_hue/vocab/en-us/DecreaseBrightnessKeyword.voc

```diff
@@ -0,0 +1,6 @@
+dim my lights
+dim the lights
+decrease the brightness of my lights
+decrease the brightness of the lights
+turn down my lights
+turn down the lights
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `DecreaseBrightnessKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.